core/domain/services/admin/privacy_policy/PrivacyPolicy.php

```diff
@@ -17,7 +17,7 @@
  * @author         Mike Nelson
  * @since          $VID:$
  */
-class PolicyGenerator implements PolicyGeneratorInterface
+class PrivacyPolicy implements PrivacyPolicyInterface
```

Comment:
please be sure to update the phpdocs for this class as well which I think are currently left at:

```php
 * Class Core
 * Description
```

Reply:
ya those aren't super descriptive, are they?